### DISCUSSION TITLE
SEO for Roam Garden

### DISCUSSION THREAD

**Discussion by ybalkind** [roam-garden/i#71] [Feature requests & Ideas] [answered]
I see no discussion or info about this. 

Are Roam Garden sites indexed and do they have respectable SEO?

**Comment by Stvad** [comment, marked as answer]
Hey @ybalkind yes the gardens are indexed.   
Also, it's built on Gatsby that usually does "good seo by default". If you have more specific questions - please lmk

**Reply by ybalkind** [reply]
Great that is all I needed to know, thanks
